modules/fedimint-mint-client/src/lib.rs

```diff
@@ -341,7 +341,11 @@ pub struct MintOperationMeta {
 #[serde(rename_all = "snake_case")]
 pub enum MintOperationMetaVariant {
     Reissuance {
-        out_point: OutPoint,
+        // Deleted in 0.3.0:
+        // # out_point: OutPoint,
+        // Introduced in 0.3.0:
+        #[serde(default)]
+        out_points: Vec<OutPoint>,
```

Comment:
This encoding is highly redundant (always the same txid), do we care?

Reply:
If it's guaranteed to always be the same txid then oftentimes such redundancies are PITA down the line, even if not a storage problem. I guess `txid` + `Vec<uX>` is simple enough to do that why not?

Reply:
Done, good point!